latex/interni/doc_interna/norme_di_progetto/res/sections/01_content.tex

```diff
@@ -118,10 +118,33 @@ \subsection{Verifica}
 Dopo aver eseguito un commit sul ramo da integrare approva la Pull Request e conferma il merge secondo le norme di progetto descritte nella sezione dedicata.
 
 \subsection{Approvazione}
-IN SOSPESO
+L'approvazione viene svolta dal Responsabile di Progetto.\\
+Si effettua questa pratica non appena il file prodotto viene verificato e passa tutte le eventuali reviews create dai verificatori e non necessita di ulteriori modifiche o revisioni future. Spetterà quindi l'ultima parola al Responsabile di Progetto che dovrà approvare la versione del file. In caso si riscontrassero ulteriori problematiche, verrà riportata la pull request nella fase di verifica citata precedentemente nel punto 2.2. \\
```

Comment:
Solo per i verbali.. per i documenti più importanti lápprovazione viene fatta prima di una consegna.
Eliminerei "e non necessita di ulteriori modifiche e revisioni future" perché non é detto che le parti verificate non vengano più toccate in futuro.

Reply:
L’approvazione viene svolta dal Responsabile di Progetto.
L'approvazione consiste nell'aprire una Pull Request di approvazione. Nel caso in cui il Responsabile di progetto riscontrasse ulteriori problematiche,  segnalerà ai Verificatori le eventuali modifiche da apportare. I verificatori apporteranno tali modifiche prima di chiudere la Pull Request di approvazione e quindi di integrare i cambiamenti nel branch documentation.
Per i verbali si effettua questa pratica non appena il file prodotto viene verificato e quindi passa tutte le eventuali reviews create dai verificatori mentre per tutti gli altri documenti prima di una consegna.

Reply:
questa potrebbe essere la versione finale